src/scrcmd.c

```diff
@@ -2129,6 +2181,13 @@ bool8 ScrCmd_playmoncry(struct ScriptContext *ctx)
     return FALSE;
 }
 
+bool8   ScrFunc_playfirstmoncry(struct ScriptContext *ctx)
+{
+    u16 species = GetMonData(GetFirstLiveMon(), MON_DATA_SPECIES);
+    PlayCry_Script(species, 0);
```

Comment:
Do we need the species variable if it only gets used once anyway?

Reply:
No I just thought it looked cleaner than having `PlayCry_Script(GetMonData(GetFirstLiveMon(), MON_DATA_SPECIES), 0);`